docker/Dockerfile.processor

```diff
@@ -12,6 +12,7 @@ COPY . /app/
 USER app
 
 RUN mkdir -p /tmp/symbols/cache
+RUN mkdir -p /tmp/symbols/tmp
```

Comment:
stackwalker requires these two directories to exist before it runs. Otherwise it says something singularly pithy like "symbols-cache and symbols-tmp must be on the same filesystem".